packages/ui/src/lib/components/discord/MessageAuthor.tsx

```diff
@@ -15,10 +17,12 @@ export function DiscordMessageAuthor({ avatar, username, bot, time }: IDiscordMe
 			/>
 			<h2 className="m-0 text-size-inherit font-medium leading-snug" id="user-info">
 				<span className="mr-1" id="username">
-					<span className="cursor-pointer text-base font-medium text-white hover:underline">{username}</span>
+					<span className={`cursor-pointer text-base font-medium hover:underline ${color ?? 'text-white'}`}>
+						{username}
+					</span>
 					{bot ? (
 						<span className="relative top-1 ml-1 rounded bg-blurple px-1 vertical-top text-xs text-white" id="bot">
-							BOT
+							{verified ? '✓ BOT' : 'BOT'}
```

Comment:
try using `svg` instead
```svg
<svg aria-hidden="false" width="16" height="16" viewBox="0 0 16 15.2">
		<path d="M7.4,11.17,4,8.62,5,7.26l2,1.53L10.64,4l1.36,1Z" fill="currentColor" />
</svg>
```

Reply:
🤷, I tried it but it doesn't change much. And I think it's a bit overkill for what we're trying to achieve here.

Reply:
What about an icon from react-icons/fi? Like for example "FiCheck"

Reply:
<img width="139" alt="Schermafbeelding 2023-04-29 om 14 54 56" src="https://user-images.githubusercontent.com/74485845/235303560-ce8fae7d-22f3-43de-ad66-c3d3a9830b67.png">
<img width="140" alt="Schermafbeelding 2023-04-29 om 14 55 04" src="https://user-images.githubusercontent.com/74485845/235303566-59288cf2-38d1-478c-bdf9-b85309976fcb.png">

Ended up using `react-icons` as Crawl said. The latter one does look a bit better, it uses FiCheck. I also experimented with `BiCheck` and `BsCheck` but they don't fit well.

Discord:
<img width="46" alt="Schermafbeelding 2023-04-29 om 14 56 10" src="https://user-images.githubusercontent.com/74485845/235303609-1ff64f27-c98b-4c16-ad86-b1b0aa9be65c.png">